client/src/components/ProfileSettingsSkeleton/ProfleSettingsLInks.tsx

```diff
@@ -0,0 +1,50 @@
+import { Typography } from '@material-ui/core';
+import { FunctionComponent } from 'react';
+import ProfilePhoto from '../../pages/ProfileSetting/ProfileSetting';
+
+// Render in place of unavailable features during development
+const Unavailable: FunctionComponent = (): JSX.Element => {
+  return <Typography>Unavailable Feature</Typography>;
+};
```

Comment:
This is a great idea! Maybe add the location to change for this (e.g. "This page doesn't exist, modify `path_to_file` once the component is implemented.")